jbpm-designer-distribution-wars/src/main/assembly/assembly-standalone-tomcat-8_0.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
 
-  <id>tomcat7.0</id>
+  <id>tomcat8.0</id>
```

Comment:
As an aside; please see the comment [here](https://github.com/psiroky/kie-wb-distributions/blob/aef6c2d784797d0a8b2c7a282713eafaa748f96a/kie-drools-wb/kie-drools-wb-distribution-wars/src/main/community/assembly-kie-drools-wb-eap-7.xml#L22).. would it be an opportune time to fix this for designer?